package com.javaguru.student_igors_jurkovs.lesson_13.level_3_junior;

class MultiCatchExample {

    public static void main(String[] args) {

        try {

        } catch (Exception e) {

        }
//        catch (ArithmeticException a) { Этот блок не нужен, потому что ошибка наследуется от Exception.
//
//        }
    }
}
